### PR TITLE
SAK-48572: Trinity Portal / Messages: Label; not in bold text

### PR DESCRIPTION
**Pull request by zhuoY121**
JIRA: https://sakaiproject.atlassian.net/browse/SAK-48572

This PR is a supplement to https://github.com/sakaiproject/sakai/pull/11472. Add the bold text for "Rename Messages Folder" page.

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgFolderAdd.jsp [hunk 1]
```diff
@@ -30,7 +30,7 @@
 			   <h:messages styleClass="alertMessage" id="errorMessages" rendered="#{! empty facesContext.maximumSeverity}" />
 			 
  <h:panelGrid styleClass="jsfFormTable" columns="2">
-			  <h:panelGroup styleClass="shorttext required">
+			  <h:panelGroup styleClass="shorttext required form-control-label">
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgFolderAdd.jsp:33]: form-control-label is from _sitemanage.scss, it is part of the site manage tool's styles, not the msgcntr styles. So, maybe you should try using bootstrap's form-label class instead?

**Review by adrianfish — CHANGES_REQUESTED** [review]
I think you should use bootstrap's form-label class here. Or, if that is impossible, move form-control-label from _sitemanage.scss to a more global part of the sass build. If you have questions about that, just ask :)

File: msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgFolderAdd.jsp (hunk 1 above)
Comment (zhuoY121) [reply msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgFolderAdd.jsp:33]: Hi Adrian, thanks for your suggestion! It seems like ```form-label``` does not work in this case, so I move ```form-control-label``` from ```_sitemanage.scss``` to ```tool.scss```. Would you mind taking a look at it?

**Review by adrianfish — APPROVED** [review]
Nice, Zhou. tool.scss is a far better place for that.

**Pull request merged**